src/mgmt/drpc_internal.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * (C) Copyright 2019 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+ * The Government's rights to use, modify, reproduce, release, perform, display,
+ * or disclose this software are subject to the terms of the Apache License as
+ * provided in Contract No. 8F-30005.
+ * Any reproduction of computer software, computer software documentation, or
+ * portions thereof marked with this legend must also reproduce the markings.
+ */
+/**
+ * ds_mgmt_drpc: Internal definitions for dRPC handlers. These functions
+ * accept a Drpc__Call, process it for input parameters, perform the desired
+ * function, and return the results as a Drpc__Response.
+ */
+
+#ifndef __MGMT_DRPC_INTERNAL_H__
+#define __MGMT_DRPC_INTERNAL_H__
+
+#include <daos/drpc.pb-c.h>
+
+void
+ds_mgmt_drpc_kill_rank(Drpc__Call *drpc_req, Drpc__Response *drpc_resp);
```

Comment:
(style)  need consistent spacing around '*' (ctx:WxV)
(style)  need consistent spacing around '*' (ctx:WxV)

Reply:
I'm unsure what this one is actually about.

Reply:
In the original output in the Jenkins logs, it's actually referring to the '\*'. Github is just interpreting it as a markdown symbol in the comment. However I think this is a bug in checkpatch, since the spacing looks correct here. I did find some old reports of checkpatch having issues with this warning when the pointer type is a typedef (as it is here).